java.md

```diff
@@ -6,6 +6,19 @@ Java is well supported and generally performant out-of-the-box on arm64. [Amazon
 
 This page includes specific details about building and tuning Java application on Graviton.
 
+### Java versions
+JDK binaries for arm64 are available from a number of
+different sources.  [Amazon Corretto](https://aws.amazon.com/corretto/) is
+continuing to improve performance of Java workloads running on Graviton processors and
+if you have the choice of a JDK to use we recommend using Corretto as it
+rovides the fastest way to get access to the performance improvements AWS is making.
+Versions of Corretto released since October 2020 are built to use the
+most optimal atomic operations within the JVM: Corretto11 (all
+variants); Correto8 (on Amazon Linux 2 only). This has shown to reduce
+GC time in some workloads.  While Java 8 is fully supported on Arm
```

Comment:
i would make the Java8 vs Java11 recommend a dedicated section and bring it up earlier in the doc, so developers will easily see it instead of one sentence in the end of another paragraph